labelbox_dev/data_row.py

```diff
@@ -55,17 +56,39 @@ def get_by_global_keys(global_keys):
     pass
 
 
-def create(dataset_id, data_row: CreateDataRowType):
+def create_one(dataset_id, data_row: CreateDataRowType):
+    # TODO: Handle row_data as local file
     create_data_row_input = {'dataset_id': dataset_id, 'data_row': data_row}
-    # TODO: upload if row_data is local file
     data_row_json = Session.post_request(f"{DATA_ROW_RESOURCE}",
                                          json=create_data_row_input)
     return DataRow(data_row_json)
 
 
-def create_many(dataset_id, data_rows: List[CreateDataRowType]):
-    # TODO: Bulk creation and handling local files
-    pass
+def create(dataset_id,
+           data_rows: List[CreateDataRowType],
+           run_async: bool = False):
+    # TODO: Handle row_data as local file
+    create_data_rows_input = {
+        'dataset_id': dataset_id,
+        'data_rows': data_rows,
+        'run_async': run_async
+    }
+    if run_async:
+        task_json = Session.post_request(f"{DATA_ROW_RESOURCE}/bulkcreate",
+                                         json=create_data_rows_input)
+        task_id = task_json.get(
+            'taskId')  # TODO: change to snake_case from backend
+
+        if task_id is None:
+            raise LabelboxError(
+                f"Failed to retrieve task information for `data_row.create()` async operation"
+            )
+        return DataRowImportTask.get_by_id(task_id)
+
+    # TODO: Need to paginate sync call. Currently, 100 data rows is the limit
+    data_rows = Session.post_request(f"{DATA_ROW_RESOURCE}/bulkcreate",
+                                     json=create_data_rows_input)
+    return [DataRow(data_row_json) for data_row_json in data_rows]
```

Comment:
Since we're only grabbing the first 100 data rows at the moment from the backend, there's no need to yield right now. I think we need to consider whether we want to introduce a paginated collection first, before implementing this return type as a generator